### ISSUE TITLE
Beast Strafing has no target

### ISSUE THREAD

**Issue by Shizuke** [HerculesWS/Hercules#1600]
As said in this topic: http://herc.ws/board/tracker/issue-8417-beast-strafing/
Beast strafing has no target and it's very difficult to use because you have to double strafe>attack>beast strafe.

Playtester suggested the following change:

> Yes, BS should be targetable. That's what I meant when I said "target".
> Sorry if I wasn't clear.
> `499,-9,8,4,-1,0,0,1,2,no,0,0x208,0,weapon,0,	HT_POWER,Beast Strafing`
> Should be:
> `499,-9,8,1,-1,0,0,1,2,no,0,0x8,0,weapon,0,	HT_POWER,Beast Strafing`

@edit: If i'm not wrong it should be:

```
>	Id: 499
> 	Name: "HT_POWER"
> 	Description: "Beast Strafing"
> 	MaxLevel: 1
> 	Range: -9
> 	Hit: "BDT_MULTIHIT"
> 	SkillType: {
> 		Enemy: true
> 	}
> 	SkillInfo: {
> 		Spirit: true
> 		NoCastSelf: true
> 	}
> 	AttackType: "Weapon"
> 	Element: "Ele_Weapon"
> 	NumberOfHits: 2
> 	SkillData1: 100
> 	CoolDown: 0
> 	Requirements: {
> 		SPCost: 12
> 		WeaponTypes: {
> 			Bows: true
> 		}
> 		AmmoTypes: {
> 			A_ARROW: true
> 		}
> 		AmmoAmount: 1
> 	}
> },
```

**Comment by Smokexyz** [comment]
If the case is `499,-9,8,1,-1,0,0,1,2,no,0,0x8,0,weapon,0, HT_POWER,Beast Strafing`
then `0x8` would be `Spirit` only, without `NoCastSelf`.
`SkillType` is 👍 
Feel free to add a PR with this change.

**Comment by Shizuke** [comment]
Hello, i can't add a PR, first time here in github i think i'm doing it wrong or maybe i don't have permissions for that.

**Comment by Smokexyz** [comment]
Merged in #1605
